cli/popper/utils.py

```diff
@@ -187,11 +188,13 @@ def b(t):
                 info('DEBUG: subprocess.Popen() with combined stdout/stderr\n')
             p = Popen(cmd, stdout=PIPE, stderr=STDOUT, shell=True,
                       universal_newlines=True)
+            popper.cli.process_list.append(p.pid)
         else:
             if debug:
                 info('DEBUG: subprocess.Popen() with separate stdout/stderr\n')
             p = Popen(cmd, stdout=outf, stderr=errf, shell=True,
                       universal_newlines=True)
+            popper.cli.process_list.append(p.pid)
```

Comment:
since this is done regardless of the value of `verbose`, it can be done outside the conditional block